modules/migration-document-attributes.adoc

```diff
@@ -0,0 +1,13 @@
+{product-author}
```

Comment:
Please move these attributes into the `common-attributes.adoc` file and include it again. I had a chat with Vikram, and he says that it would be more appropriate to add the new attributes there than to create a new one. Aligned products can make extra attributes files, but Vikram says that migrating doesn't count as an aligned product.

Reply:
OK

Reply:
In that case, all the CAM rebranding stuff will also be moved there. :-)